### PR TITLE
FE-94: Update contact links

### PR DESCRIPTION
**Pull request by rajumsys**
There isn't a specific category for general compliance issues like DKIM validation issue. But as they're grouped under technical support, i've linked to technical issue.

### CODE REVIEW

File: articles/getting-started/how-to-get-help.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ Find out whether the service is up, down, sideways, on vacation or going slow by
...
 | **Upgrades - Large Plans & Add-ons**            | **[Contact Sales](mailto:insidesales@sparkpost.com?subject=Account%20Change)**                                                                                   |
-| **Cancel your Service**                         | **[Submit a support request](https://www.sparkpost.com/submit-a-ticket?email[subject]=Web:%20Cancel%20Service%20Request:%20Selfserve%20Account)** |
-| **Cancel your Service - Large Plans >2M/month** | **[Submit a support request](https://www.sparkpost.com/submit-a-ticket?email[subject]=Web:%20Cancel%20Service%20Request:%20Large%20Account)**     |
-| **Billing Inquiries**                           | **[Contact our Billing Team](mailto:billing@sparkpost.com?subject=Billing%20Request)**           |
+| **Cancel your Service**                         | **[Submit a support request](https://app.sparkpost.com/dashboard/?supportTicket=true&supportIssue=account_cancellation)** |
+| **Cancel your Service - Large Plans >2M/month** | **[Submit a support request](https://app.sparkpost.com/dashboard/?supportTicket=true&supportIssue=account_cancellation)**     |
```
Comment (ewandennis) [inline articles/getting-started/how-to-get-help.md]: Do we still need 2 entries for "cancel your service" ? They both go to the same support form now so...

**Comment by ewandennis** [comment]
We should also fix up `billing/upgrading-your-account.md`

File: articles/getting-started/how-to-get-help.md (hunk 1 above)
Comment (mitchwh24) [reply articles/getting-started/how-to-get-help.md]: we dont need the 2 now - we used to run a different process on the large ones

**Comment by ewandennis** [comment]
@rajumsys For this:
> Use some more app-centric language for in-app support links?

...the idea was to make it clear that those in-app support links were deep links into the app. Not important though. The current text looks ok.

**Pull request merged**